### PR TITLE
Release v0.16.2

### PR DESCRIPTION
**Pull request by ueco-jb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+
+[Full Changelog](https://github.com/CosmWasm/cw-multi-test/compare/0.16.1...v0.16.2)
+
+**Merged pull requests:**
+
+- Update `cw-utils` and `cw-storage-plus` to 1.0 [\#15](https://github.com/CosmWasm/cw-multi-test/pull/15) ([ueco-jb](https://github.com/ueco-jb))
+
+## [0.16.1](https://github.com/CosmWasm/cw-multi-test/tree/0.16.1) (2022-11-23)
+
+[Full Changelog](https://github.com/CosmWasm/cw-multi-test/compare/v0.16.1...0.16.1)
```
Comment (webmaster128) [inline CHANGELOG.md:13]:
```suggestion
[Full Changelog](https://github.com/CosmWasm/cw-multi-test/compare/v0.16.0...v0.16.1)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
 
+## [v0.16.2](https://github.com/CosmWasm/cw-multi-test/tree/v0.16.2) (2022-12-15)
+
+[Full Changelog](https://github.com/CosmWasm/cw-multi-test/compare/0.16.1...v0.16.2)
+
+**Merged pull requests:**
+
+- Update `cw-utils` and `cw-storage-plus` to 1.0 [\#15](https://github.com/CosmWasm/cw-multi-test/pull/15) ([ueco-jb](https://github.com/ueco-jb))
+
+## [0.16.1](https://github.com/CosmWasm/cw-multi-test/tree/0.16.1) (2022-11-23)
```
Comment (webmaster128) [inline CHANGELOG.md:11]:
```suggestion
## [v0.16.1](https://github.com/CosmWasm/cw-multi-test/tree/v0.16.1) (2022-11-23)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [v0.16.2](https://github.com/CosmWasm/cw-multi-test/tree/v0.16.2) (2022-12-15)
+
+[Full Changelog](https://github.com/CosmWasm/cw-multi-test/compare/0.16.1...v0.16.2)
```
Comment (webmaster128) [inline CHANGELOG.md:5]:
```suggestion
[Full Changelog](https://github.com/CosmWasm/cw-multi-test/compare/v0.16.1...v0.16.2)
```

**Review by webmaster128 — APPROVED** [review]
Thanks!

**Comment by Tofel** [comment]
@ueco-jb when can I expected the `0.16.2` to be published to crates.io?

**Comment by ueco-jb** [comment]
@Tofel just about now.

**Comment by Tofel** [comment]
@ueco-jb great, you're very responsive guys, that's great!

**Pull request merged**